src/components/builder/Stats/Defense.jsx

```diff
@@ -0,0 +1,78 @@
+import { View, Text, Image, ScrollView } from "react-native"
+import { styles } from "./StatsStyle"
+
+function Defense({ health, stamina, baseDefense, maxDefense, augDefense, resFire, resWater, resIce, resThunder, resDragon }) {
+    return (
+        <ScrollView
+            showsHorizontalScrollIndicator={false}
+            showsVerticalScrollIndicator={false}
+        >
+            <View style={styles.statsOn}>
+                <View style={styles.leftView}>
+                    <Image source={require("../../../assets/images/icons/health-icon.png")} style={styles.leftViewImage} />
+                    <Text style={styles.text}>Health : </Text>
+                </View>
+                <Text style={styles.text}>{health}</Text>
+            </View>
+
+            <View style={styles.statsOff}>
+                <View style={styles.leftView}>
+                    <Image source={require("../../../assets/images/icons/stamina-icon.png")} style={styles.leftViewImage} />
+                    <Text style={styles.text}>Stamina : </Text>
+                </View>
+                <Text style={styles.text}>{stamina}</Text>
+            </View>
+
+            <View style={styles.statsOn}>
+                <View style={styles.leftView}>
+                    <Image source={require("../../../assets/images/icons/defense-icon.png")} style={styles.leftViewImage} />
+                    <Text style={styles.text}>Defense : </Text>
+                </View>
+                <Text style={styles.text}>{baseDefense} | {maxDefense} | {augDefense}</Text>
+            </View>
+
+            <View style={styles.statsOff}>
+                <View style={styles.leftView}>
+                    <Image source={require("../../../assets/images/icons/fire-icon.png")} style={styles.leftViewImage} />
+                    <Text style={styles.text}>Fire Resistance : </Text>
+                </View>
+                <Text style={styles.text}>{resFire}</Text>
+            </View>
+
+            <View style={styles.statsOn}>
```

Comment:
Y aurait-il moyen de transformer tout ça en un composant statsOn vs StatOff ou même un seul et meme composant à qui tu passerais le of / off et le contenu ? On a énormément de répétitions ici